backend/server.ts

```diff
@@ -1,6 +1,7 @@
 import express, { Request, Response, NextFunction } from "express";
 import User from "./models/user";
 const app = express();
+const mongoose = require('mongoose');
```

Comment:
looks like we won't be using mongoose for this file yet, so remove the require statement for now